lib/logger/src/styles.ts

```diff
@@ -16,9 +21,22 @@ export const styles = {
   dim: colours.grey,
   title: colours.bold.underline,
   taskHeader: colours.bgWhite.black,
-  errorHighlight: colours.red,
-  error: (string: string): string => `${styles.errorHighlight.bold('‼︎')} ${styles.title(string)}`,
-  warningHighlight: colours.yellow,
-  warning: (string: string): string => styles.warningHighlight.bold('⚠︎') + ' ' + string,
-  ruler: (): string => styles.dim('─'.repeat(process.stdout.columns / 2))
+  errorHighlight: colours.bgRed.bold,
+  error: (string: string): string => `${styles.errorHighlight(' × ️')} ${styles.title(string)}`,
+  warningHighlight: colours.bgYellow.bold,
+  warning: (string: string): string => styles.warningHighlight(' ! ') + ' ' + string,
+  infoHighlight: colours.bgBlueBright.bold,
+  info: (string: string): string => styles.infoHighlight(' i ') + ' ' + string,
+  helpHighlight: colours.bgGreen.bold,
+  help: (string: string): string => styles.helpHighlight(' ? ') + ' ' + string,
+  ruler: (): string => styles.dim('─'.repeat(process.stdout.columns / 2)),
+  box: (string: string, options: Partial<BoxenOptions>) =>
+    boxen(string, {
+      borderStyle: 'round',
+      padding: { left: 1, right: 1 },
+      ...options
+    }),
+  groupHeader: (string: string) => ` ╭─${'─'.repeat(stripAnsi(string).length)}─╮
```

Comment:
**issue**: i get misaligned headers if my terminal window is too small (i was testing the code in a tmux pane 80 characters wide.) i suspect this is a bug in `boxen` as i can't see why else the space at the beginning of the string would be trimmed.

![Screenshot 2024-04-22 at 16 29 17](https://github.com/Financial-Times/dotcom-tool-kit/assets/7020796/43060c4e-cafe-4b14-a228-551f8612bb1e)

Reply:
it looks like it's trimming all the leading whitespace! even from the `?`/`i`. the version of `boxen` i'm using is a fork of an older version of Sindre's library that still seems vaguely maintained. i don't know if the current Sindre version would work better but it's ESM only 🙃